homeassistant/components/webostv/notify.py

```diff
@@ -46,8 +46,8 @@ async def async_send_message(self, message: str = "", **kwargs: Any) -> None:
             if not self._client.is_connected():
                 await self._client.connect()
 
-            data = kwargs.get(ATTR_DATA, {})
-            icon_path = data.get(CONF_ICON)
+            data = kwargs.get(ATTR_DATA)
```

Comment:
For this PR to be valid at all, this is always present and you should then use `kwargs[ATTR_DATA]`

Reply:
Or even better add it to the signature `data: dict[str, Any] | None = None,`

Reply:
I think some linters may complain if we change the signature compared to the parent method. Let's explore the suggested improvements. 👍 

This PR makes sure we don't error if the value is None.

Reply:
Clean up here:
https://github.com/home-assistant/core/pull/66803